packages/lesswrong/components/common/LoadMore.tsx

```diff
@@ -98,7 +98,7 @@ const LoadMore = ({
    * a non-zero count that graphql cached during SSR.
    */
   const isFirstRender = useIsFirstRender();
-  loading = loading && !(isFirstRender && (count ?? 0) > 0);
+  loading = loading && !isFirstRender;
```

Comment:
I feel like the `LoadMore` component should only show a loading animation after it's clicked, and not when it's first rendered. Let me know if you disagree.

Reply:
This seems kind-of scary to me, but I after a quick look I couldn't find anywhere that actually breaks so maybe it's fine - we should just keep an eye out for missing loaders in the future.

We should also update the comment above though as it's now out of date with the code.